src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/CreateOrganizationContext.tsx

```diff
@@ -0,0 +1,309 @@
+//     priority: 3,
+//     providerId: 'AWS',
+//     providerName: 'Amazon Web Services',
+//     regionId: 'us-west-2-2',
+//     regionName: 'US West (Oregon)',
+//   },
+// ]
+// const FAKE_AZURE: Cluster[] = [
+//   {
+//     isBeta: false,
+//     isProviderAvailable: true,
+//     isRegionAvailable: true,
+//     priority: 1,
+//     providerId: 'Azure',
+//     providerName: 'Microsoft Azure',
+//     regionId: 'eastus',
+//     regionName: 'Virginia',
+//   },
+//   {
+//     isBeta: false,
+//     isProviderAvailable: true,
+//     isRegionAvailable: true,
+//     priority: 2,
+//     providerId: 'Azure',
+//     providerName: 'Microsoft Azure',
+//     regionId: 'westeurope',
+//     regionName: 'Amsterdam',
+//   },
+// ]
+// const FAKE_GCP: Cluster[] = [
+//   {
+//     isBeta: false,
+//     isProviderAvailable: true,
+//     isRegionAvailable: true,
+//     priority: 1,
+//     providerId: 'GCP',
+//     providerName: 'Google Cloud',
+//     regionId: 'eu-west1',
+//     regionName: 'Belgium',
+//   },
+//   {
+//     isBeta: false,
+//     isProviderAvailable: true,
+//     isRegionAvailable: true,
+//     priority: 2,
+//     providerId: 'GCP',
+//     providerName: 'Google Cloud',
+//     regionId: 'us-central1',
+//     regionName: 'Iowa',
+//   },
+//   {
+//     isBeta: false,
+//     isProviderAvailable: true,
+//     isRegionAvailable: true,
+//     priority: 98,
+//     providerId: 'GCP',
+//     providerName: 'Google Cloud',
+//     regionId: 'us-central1-2',
+//     regionName: 'Iowa 2',
+//   },
+//   {
+//     isBeta: false,
+//     isProviderAvailable: true,
+//     isRegionAvailable: true,
+//     priority: 99,
+//     providerId: 'GCP',
+//     providerName: 'Google Cloud',
+//     regionId: 'us-west1',
+//     regionName: 'Oregon',
+//   },
+// ]
+// const FAKE_CLUSTERS: Cluster[] = [...FAKE_AWS, ...FAKE_AZURE, ...FAKE_GCP]
+
+export const CreateOrgProvider: FC<Props> = ({children}) => {
+  const [orgName, setOrgName] = useState(DEFAULT_CREATE_ORG_CONTEXT.orgName)
+  const [clusters, setClusters] = useState<ClusterMap>(
+    DEFAULT_CREATE_ORG_CONTEXT.clusters
+  )
+  const [currentProvider, setCurrentProvider] = useState<ProviderID>(
+    DEFAULT_CREATE_ORG_CONTEXT.currentProvider
+  )
+  const [currentRegion, setCurrentRegion] = useState<string>(
+    DEFAULT_CREATE_ORG_CONTEXT.currentRegion
+  )
+  const [createOrgLoadingStatus, setCreateOrgLoadingStatus] =
+    useState<RemoteDataState>(RemoteDataState.NotStarted)
+
+  const changeCurrentProvider = useCallback(
+    (provider: ProviderID) => {
+      setCurrentProvider(provider)
+    },
+    [setCurrentProvider]
+  )
+
+  const changeCurrentRegion = useCallback(
+    (region: string) => {
+      setCurrentRegion(region)
+    },
+    [setCurrentRegion]
+  )
+
+  const createOrg = useCallback((): Promise<void> => {
+    if (createOrgLoadingStatus === RemoteDataState.Loading) {
+      return new Promise(resolve => resolve(null))
+    }
```

Comment:
What's the purpose of this `if` check for the remote data state being `Loading`?